app/src/main/java/com/example/coinage/fragments/AddFragment.java

```diff
@@ -31,9 +32,9 @@
 public class AddFragment extends Fragment {
     public static final String TAG = "AddFragment";
 
-    public static final Calendar myCalendar = Calendar.getInstance();
+    public final Calendar myCalendar = Calendar.getInstance();
```

Comment:
Nice!

More details on static variables and static keyword may be found here:
https://www.edureka.co/blog/static-keyword-in-java/#StaticVariable

Just food for thought:
Static keyword essentially creates a globally unique instances -- this may lead to issues later if data needs to be properly encapsulated -- and may be even buggy if the instance is written+read from multiple threads ...
so you need to use them only when you truly need a single global instance especially like constant strings and constant numbers which are globally unique readonly objects.

Reply:
Sounds good, I'll keep that in mind!